sdk/storage/azure-storage-files-shares/src/share_responses.cpp

```diff
@@ -40,12 +39,12 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
   {
     while (true)
     {
-      auto rawResponse = PollInternal(context);
+      auto rawResponse = Poll(context);
 
       if (m_status == Azure::Core::OperationStatus::Succeeded)
       {
         return Azure::Response<Models::GetShareFilePropertiesResult>(
-            m_pollResult, std::move(rawResponse));
+            m_pollResult, std::make_unique<Azure::Core::Http::RawResponse>(rawResponse));
```

Comment:
```suggestion
            m_pollResult, std::make_unique<Azure::Core::Http::RawResponse>(std::move(rawResponse)));
```

Reply:
Is it possible to avoid a copy here?

Reply:
Sure, but your are leaving the Operation useless